src/arche/rules/category.py

```diff
@@ -30,12 +30,13 @@ def get_difference(
             .fillna(0)
             .sort_values(by=[source_key, target_key], kind="mergesort")
         )
-        cats.name = f"Coverage difference in {c}"
+        cats.name = f"Coverage for {c}"
         result.stats.append(cats)
         cat_difs = ((cats[source_key] - cats[target_key])).abs()
-        cat_difs.name = (
-            f"Coverage difference between {source_key}'s and {target_key}'s {c}"
-        )
+        cat_difs = cat_difs[cat_difs > 10]
```

Comment:
Maybe instead of a hardcoded value there could be a default value and option to be override it. 
We have something similar in BVATF: 

https://github.com/scrapinghub/bv_qa_automation/wiki/Coverage-Compare-Fields-Limits:





Brand_name |   | 10% |   |  
Description |   |   | 15% |  
Features |   |   | 15% |  
Identifier_add | 5% |   |  

Reply:
Once anyone asks for that, it could be done.